### PR TITLE
fix: prevent rich text references from opening on click

### PR DESCRIPTION
**Pull request by m10l**
When clicking on reference cards within the rich text editor, the entry is immediately opened. This PR changes that so that clicking on a reference card only selects the card which allows the user to then press backspace to remove the referenced entry from the rich text editor field.

### CODE REVIEW

File: packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx [hunk 1]
```diff
@@ -93,7 +98,9 @@ export function WrappedEntryCard(props: WrappedEntryCardProps) {
 
   return (
     <EntryCard
-      href={props.getEntryUrl ? props.getEntryUrl(props.entry.sys.id) : undefined}
+      href={
+        props.getEntryUrl && props.isClickable ? props.getEntryUrl(props.entry.sys.id) : undefined
```
Comment (djagya) [inline packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx:102]: I think the same should be done for assets if we want to be consistent on how both cards are handled?

i.e. `packages/rich-text/src/plugins/EmbeddedEntityBlock/FetchingWrappedAssetCard.js` and `packages/reference/src/assets/WrappedAssetCard/WrappedAssetCard.tsx`

Comment (djagya) [inline packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx:102]: I think the same should be done for assets if we want to be consistent on how both cards are handled?

i.e. `packages/rich-text/src/plugins/EmbeddedEntityBlock/FetchingWrappedAssetCard.js` and `packages/reference/src/assets/WrappedAssetCard/WrappedAssetCard.tsx`

File: packages/rich-text/src/plugins/EmbeddedEntityBlock/FetchingWrappedEntryCard.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ export function FetchingWrappedEntryCard(props) {
       entry={entry}
       onEdit={props.onEdit}
       onRemove={props.onRemove}
+      isClickable={false}
```
Comment (djagya) [inline packages/rich-text/src/plugins/EmbeddedEntityBlock/FetchingWrappedEntryCard.js:55]: Should also be applied to `FetchingWrappedAssetCard.js` IIUC

File: packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx (hunk 1 above)
Comment (m10l) [reply packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx:102]: Ah good catch, I forgot to check assets 🤦

Comment (m10l) [reply packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx:102]: Updated assets, I'll mark this and the other comment as resolved

**Pull request merged**